### DISCUSSION TITLE
Добро пожаловать в чат по обсуждению opencart3-exchange1c !

### DISCUSSION THREAD

**Discussion by Tseka85** [Tseka85/opencart3-exchange1c#11] [General]
## 👋 Добро пожаловать!
  Мы используем обсуждения как место для общения с другими членами нашего сообщества.
  * Задавайте вопросы, которые вас интересуют.
  * Делитесь идеями.
  * Общайтесь с другими членами сообщества.
  * Помните, что это сообщество, которое мы
  строить вместе 💪.

  Чтобы начать, оставьте комментарий ниже, представив себя и расскажите нам о том, что вы делаете с этим сообществом.

**Comment by rddomino** [comment]
Добрый день, больше всего интересует остатки на складах, обмен именно остатков, в описание такого пункта не вижу. И как-то управлять цена можно? Например в базе оптовые цены, а в магазине на сайте с наценкой, и чтобы накладная выбивалась с ценами по сайту, или отдельную базу надо под такое заводить?

**Reply by Tseka85** [reply]
Этот модуль для обмена  товарами и заказами между 1С и вашим сайтом на Opencart . Я вообще ничего не понял , к чему вы это написали ? Причём тут накладные ? Ценами управляйте отдельным модулем , которых сейчас очень много .

**Reply by rddomino** [reply]
Вопрос по ценам - это второй был, он тут лишний) Остатки на складе он передает / обновляет?

**Reply by Tseka85** [reply]
Остатки обновляет .

**Reply by rddomino** [reply]
На opencart-3.0.3.2 не видит в модулях (

**Reply by Tseka85** [reply]
На этой версии не тестировался .

**Reply by AlexandrGerasimenko** [reply]
Здравствуйте, подскажите пожалуйста - в дропдауне с версиями 1С отсутствует Управление торговлей для Украины 2.3.
https://scrnli.com/yT79EBJ8ikn4zv

**Reply by Tseka85** [reply]
> Здравствуйте, подскажите пожалуйста - в дропдауне с версиями 1С отсутствует Управление торговлей для Украины 2.3.
> https://scrnli.com/yT79EBJ8ikn4zv

Выбирайте У.Т. 10.3

**Reply by AlexandrGerasimenko** [reply]
Спасибо за МЕГАоперативный ответ)
Они полностью совместимы?

**Reply by Tseka85** [reply]
> Спасибо за МЕГАоперативный ответ)
> Они полностью совместимы?

Да

**Reply by AlexandrGerasimenko** [reply]
@Tseka85 
Подскажите пожалуйста:
при ручной загрузке файлов - все ok, но при попытке законектится из 1C - 301 ошибка.
Не могу это побороть, гуглил - нахожу подобные темы 6-ти летней давности.

**Reply by Tseka85** [reply]
Выгрузка из 1С используете стандартную ?

**Reply by Tseka85** [reply]
> @Tseka85 
> Подскажите пожалуйста:
> при ручной загрузке файлов - все ok, но при попытке законектится из 1C - 301 ошибка.
> Не могу это побороть, гуглил - нахожу подобные темы 6-ти летней давности.


Вставьте в браузерную строку 

https://ВАШ_САЙТ/export/exchange1c.php

Какой будет ответ в браузере ?

**Reply by AlexandrGerasimenko** [reply]
> > @Tseka85
> > Подскажите пожалуйста:
> > при ручной загрузке файлов - все ok, но при попытке законектится из 1C - 301 ошибка.
> > Не могу это побороть, гуглил - нахожу подобные темы 6-ти летней давности.
> 
> Вставьте в браузерную строку
> 
> [https://ВАШ_САЙТ/export/exchange1c.php](https://%D0%92%D0%90%D0%A8_%D0%A1%D0%90%D0%99%D0%A2/export/exchange1c.php)
> 
> Какой будет ответ в браузере ?

Запрашиваемая страница не найдена!

**Reply by Tseka85** [reply]
> 
> 
> 
> > > @Tseka85
> > > Подскажите пожалуйста:
> > > при ручной загрузке файлов - все ok, но при попытке законектится из 1C - 301 ошибка.
> > > Не могу это побороть, гуглил - нахожу подобные темы 6-ти летней давности.
> > 
> > Вставьте в браузерную строку
> > 
> > [https://ВАШ_САЙТ/export/exchange1c.php](https://%D0%92%D0%90%D0%A8_%D0%A1%D0%90%D0%99%D0%A2/export/exchange1c.php)
> > 
> > Какой будет ответ в браузере ?
> 
> Запрашиваемая страница не найдена!
> 



Надеюсь , вы заменили слова ВАШ_САЙТ на свой сайт ?

**Reply by AlexandrGerasimenko** [reply]
Да, конечно.

**Reply by AlexandrGerasimenko** [reply]
Там даже в модуле есть ссылка на эту страницу, результат аналогичен.
Только в модуле http показывает вместо https.

**Reply by Tseka85** [reply]
> Там даже в модуле есть ссылка на эту страницу, результат аналогичен.
> Только в модуле http показывает вместо https.

У вас сайт на http или https ?

**Reply by Tseka85** [reply]
> > Там даже в модуле есть ссылка на эту страницу, результат аналогичен.
> > Только в модуле http показывает вместо https.
> 
> У вас сайт на http или https ?

Напишите мне в телеграмм .
